casper/src/main/scala/coop/rchain/casper/BlockCreator.scala

```diff
@@ -64,13 +64,14 @@ object BlockCreator {
                  #}
                  #
               """.stripMargin('#'),
-              System.currentTimeMillis(),
               accounting.MAX_VALUE,
               sec = privateKey
-            )
-            Cell[F, CasperState].modify { s =>
-              s.copy(deployHistory = s.deployHistory + deploy)
-            }
+            ) >>= (
```

Comment:
I don't like this mixture of for-comprehension and the >>= operator. 

You could do this for better readability:
```scala
slashingDeploy <- invalidLatestMessages.values.toList.traverse { invalidBlockHash =>...}
_ <- Cell[F,CasperState].modify{...}
...
```

Reply:
Done :)